tests/framework/installer/ceph_manifests_v1.0.go

```diff
@@ -206,7 +206,7 @@ spec:
 }
 
 // GetRookOperator returns rook Operator manifest
-func (m *CephManifestsV1_0) GetRookOperator(namespace string) string {
+func (m *CephManifestsV1_0) GetRookOperator(namespace string, enableCSI bool) string {
```

Comment:
This file ceph_manifests_v1.0.go is only called by the upgrade_test.go. We shouldn't make any changes to this file to make the 1.0 install the same as the 1.0 release. Since the method signature changed, the only change in this file should be the method signature. The new parameter will basically be ignored for the upgrade test when these 1.0 methods are called.